apps/site/src/views/Home/sections/FAQ/FAQ.module.scss

```diff
@@ -0,0 +1,127 @@
+@use "zothacks-theme" as theme;
+@use "bootstrap-utils" as utils;
+
+.faq {
+	position: relative;
+	display: flex;
+	flex-direction: column;
+	width: 80%;
```

Comment:
Issue: constraining the width here limits the available space on mobile views. We could instead rely on the parent container sizing, but we'll revisit this.

Reply:
Hi, I was originally using the bootstrap Container component but had to switch off to this once we made the FAQ a server component